plugins/modules/mso_schema_site_external_epg.py

```diff
@@ -34,6 +34,18 @@
     description:
     - The L3Out associated with the external epg.
     - Required when site is of type on-premise.
+    - NOTE - In NDO versions over 4.2, 
```

Comment:
Can we remove the 'NOTE -' and just include the point as is?

I think the rest looks good.

Reply:
done